docs/src/adoc/index.adoc

```diff
@@ -5194,7 +5194,7 @@ Just tell us about your types by installing the plugin and configuring your `Imm
 
 `jdbi.getConfig(JdbiImmutables.class).registerImmutable(MyValueType.class)`
 
-The configuration will both register appropriate `RowMapper`&#8288;s as well as configure the new `bindPojo` (or `@BindPojo`) binders:
```

Comment:
this is the "word joiner" unicode that pulls the `RowMapper` and the s back together.

Reply:
cleeeeeever :)